libobs/obs-encoder.c

```diff
@@ -1248,23 +1249,36 @@ static inline void send_packet(struct obs_encoder *encoder,
 void full_stop(struct obs_encoder *encoder)
 {
 	if (encoder) {
+		if (encoder->is_full_stop) {
+			return;
+		}
+		encoder->is_full_stop = true;
+
 		pthread_mutex_lock(&encoder->outputs_mutex);
 		for (size_t i = 0; i < encoder->outputs.num; i++) {
 			struct obs_output *output = encoder->outputs.array[i];
-			obs_output_force_stop(output);
+			//If checked "Enable Replay Buffer" from setting page, but not start it, don't invoke stop function
```

Comment:
I want to recommend folding at 80 columns and use the block comment (`/* */`).

Reply:
Resolved